internal/pkg/api/server/buffer/extension/dependencies/dependencies.go

```diff
@@ -0,0 +1,97 @@
+// Package dependencies contains extension to inject dependencies to the service endpoint handlers.
+package dependencies
+
+import (
+	"path/filepath"
+	"strings"
+
+	"goa.design/goa/v3/codegen"
+	"goa.design/goa/v3/eval"
+)
+
+const PkgPath = "github.com/keboola/keboola-as-code/internal/pkg/api/server/buffer/dependencies"
+
+// nolint: gochecknoinits
+func init() {
+	codegen.RegisterPluginFirst("api-dependencies", "gen", nil, generate)
+}
```

Comment:
Toto extension by mohlo ist do `common`, s tym, ze `PkgPath` by nebola konstanta ale parameter.
Spravil by si nejaku funkciu `dependencies.RegisterPlugin(pkgPath string)`.
Nebol by ziadny `init()` ... ale zavolal by si to manualne  na zaciatku `design.doc` pre buffer a templates API.

Pisal si, ze su tam rozdieliy tak mozno mi nieco uslo, ale rozdiel vidim v `pkgPath`.
Inak zostava, ze mame nejaky interface na public a iny na project dependencies (ktore vyzaduju token).

Reply:
Zkusil jsem to, ale z nějakého důvodu to nefunguje. https://haste.zneix.eu/ahutuqekec.go - tenhle soubor mám teda v `common/extension/dependencies/dependencies.go`. do `api/buffer/design.go` to importnu a dám tam:
```go
func init() {
  dependencies.RegisterPlugin("github.com/keboola/keboola-as-code/internal/pkg/api/server/buffer/dependencies")
}
```
Vygenerovaný soubory mají v sobě errors, protože tam není ten vygenerovaný kód. Když ale změním jen tu init funkci na
```go
func init() {
  dependencies.RegisterPluginStatic()
}
```
Tak vše funguje. Podle toho, co vidím, tak je ten kód identický, jen v jednom případě to používá closures a v druhém ne. Neviš, co může být problém? Mám poslat rozbitý commit?

Reply:
nemusis to davat do `init`, ... pozriem sa na to, mozno som nieco nedomyslel,
jj, prosim posli ten commit do nejakej inaj branche a tu hod link, dik

Reply:
https://github.com/keboola/keboola-as-code/pull/949